Dockerfile

```diff
@@ -49,6 +49,21 @@ COPY . .
 
 RUN make -s clean && make -s -j$(nproc) install
 
+# Now the "docs-template-testing" image, to check if the docs are up to date
+FROM --platform=${TARGETPLATFORM} build as docs-template-testing
```

Comment:
Does this means we run the docs-template-testing bits each time we want to run a test case (using e.g. `make tests/pagila`) on our local systems?

Reply:
No. Docker is clever enough to skip build stages that are not used. All thanks to the default backend (since last February) BuildKit for Docker.

See the list at the beginning of the page for [BuildKit](https://docs.docker.com/build/buildkit/)

Reply:
I see. I think I would still prefer it if we had a separate Dockerfile for the docs, does it make sense to split things that way?

Reply:
Docker does a poor job managing dependencies across multiple Dockerfiles. I created a new `Dockerfile.tests` file with a copy of our build stage so that we can hit docker build cache.